tests/foreman/cli/test_setting.py

```diff
@@ -217,10 +221,23 @@ def test_positive_update_email_reply_address(self):
 
         :caseimportance: low
 
-        :caseautomation: notautomated
+        :caseautomation: automated
         """
+        for email in valid_emails_list():
+            with self.subTest(email):
+                # The email must be escaped because some characters to not fail
+                # the parsing of the generated shell command
+                escaped_email = email.replace(
+                    '"', r'\"').replace('`', r'\`')
+                Settings.set({
+                    'name': "email_reply_address",
+                    'value': escaped_email})
+                email_reply_address = Settings.list({
+                    'search': 'name=email_reply_address'
+                })[0]
+                email = email.replace(r'"', '')
```

Comment:
why we need to replace the " char ? I was supposing that email should be saved with that char

Reply:
```bash
[root@xxx]# hammer settings set  --name=email_reply_address --value \"jjjsjs\"@example.com
Setting [email_reply_address] updated to ["jjjsjs"@example.com].
```
```bash
[root@xxx]# hammer settings list --search  name=email_reply_address
--------------------|----------------------|-------------------------------------------------------
NAME                | VALUE                | DESCRIPTION                                           
--------------------|----------------------|-------------------------------------------------------
email_reply_address | "jjjsjs"@example.com | Email reply address for emails that Foreman is sending
--------------------|----------------------|-------------------------------------------------------
[root@xxx]# 
```

Reply:
thanks @ldjebran for comment.
I will again look into it. 
I am marking it as stubbed till i have a fix solution for it.

Reply:
@jameerpathan111 : Hey, please don't stub the test. If I understand correctly the issue is that when you save the value in variable the quotes automatically get removed and test failed while assertion. To get rid of failure, you used the above statement `email = email.replace(r'"', '')`

@ldjebran : do you see what's causing the quotes to be removed from email when saving to a variable ?

Reply:
I looked into it but, I was not able to get what's causing this to happen

Reply:
now i get it, basically csv output format was causing this to happen.
If we use json format it works well.
thank you @ldjebran for looking into it.
I will do the changes you have suggested.

Reply:
@ldjebran :+1: